frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataSidebar.jsx

```diff
@@ -153,7 +153,7 @@ function DatasetFieldMetadataSidebar({
       setShouldAnimateFieldChange(true);
       // setTimeout is required as form fields are rerendered pretty frequently
       setTimeout(() => {
-        displayNameInputRef.current?.select();
+        displayNameInputRef.current?.select?.();
```

Comment:
Not sure if this is related, but I noticed a scripting error caused by the `select` property being undefined.